### ISSUE TITLE
Local constant unpicking scope

### ISSUE THREAD

**Issue by liach** [FabricMC/unpick#12]
Currently, it seems that unpick handles constant unpicking globally. This isn't bad as things like nbt types or world event ids indeed are accessed pretty much exclusively with constant values.

However, there are other more delicate cases, such as a constant (used as some easily toggled preconfigured value) for max nbt size from a private static field. Such constants currently are replaced by targetting all calls to a specific method, but this is a waste of resource as the constant usage is usually restricted to within one class, or even 1 method.

Long story short, we just want a way to specify that a constant usage is restricted to one method/class than the whole classpath in order to avoid inadvertent bugs.
